src/Raven.Server/Documents/Indexes/MapIndexBase.cs

```diff
@@ -32,10 +32,11 @@ protected override IIndexingWork[] CreateIndexWorkExecutors()
 
         public override IDisposable InitializeIndexingWork(TransactionOperationContext indexContext)
         {
-            if (sizeof(int) == IntPtr.Size || DocumentDatabase.Configuration.Storage.ForceUsing32BitsPager)
-                return null;
+            var mode = sizeof(int) == IntPtr.Size || DocumentDatabase.Configuration.Storage.ForceUsing32BitsPager
```

Comment:
What happens if I take the db from 32 bit machine (with no special configuration), then load it on 64 bits machine?
Wouldn't this result in us getting the wrong size for the bloom filter?